state-chain/pallets/cf-emissions/src/lib.rs

```diff
@@ -308,6 +308,36 @@ impl<T: Config> Pallet<T> {
 			block_number.saturated_into(),
 		));
 	}
+
+	fn burn_flip_network_fee() {
+		let flip_to_burn = T::FlipToBurn::take_flip_to_burn();
```

Comment:
This is a bug and it's not caught in the test because the mock always returns the same value of `FLIP_TO_BURN`. If we take here and then error, we're going to remove the storage but not actually burn anything, and on the next attempt we'd be starting from 0, so we'd never burn anything

Reply:
We should ofc fix the test too

Reply:
👌  good catch